.github/workflows/build.yml

```diff
@@ -37,13 +37,17 @@ jobs:
       - name: Build
         run: |
           make clean
-          make build-linux
-          make package-linux
+          make build
+          make package
       - uses: actions/upload-artifact@v2
         with:
           name: lifecycle-linux-x86-64
           path: out/lifecycle-v*+linux.x86-64.tgz
-  test-and-build-windows:
+      - uses: actions/upload-artifact@v2
+        with:
```

Comment:
Is it possible to upload multiple artifacts under one `uses: actions/upload-artiface@v2`?

Reply:
Skimming the docs, it appears the answer is no, not if we want them attached as separate artifacts, rather than a single zip containing both artifacts.